media/js/mozorg/contact-spaces.js

```diff
@@ -160,6 +160,7 @@
                     // Hide spaces and community nav+meta
                     mozMap.toggleNav('contact');
                     mozMap.toggleMobileNav('contact');
+                    mozMap.setMapState();
```

Comment:
Yeah, it seemed like when you switch from a specific space, for example, back to contact us, the map does not change. Perhaps it is just loading really slowly for me?